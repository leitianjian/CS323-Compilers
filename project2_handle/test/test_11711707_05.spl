int test0() {
    return test1(a);
}

int test1(int n){
    if(n < 0){
        return test0(1, 3);
    }else{
        char c = test0(1, 2, 3);
        return test0(c, 2);
    }
    return 1;
}