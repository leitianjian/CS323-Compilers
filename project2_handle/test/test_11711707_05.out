Error type 1 at Line 2: a variable is used without definition
Error type 2 at Line 2: test1 function is invoked without deﬁnition
Error type 8 at Line 2: function return type is not matched
Error type 9 at Line 7: the function’s arguments mismatch the declared parameters
Error type 8 at Line 7: function return type is not matched
Error type 9 at Line 9: the function’s arguments mismatch the declared parameters
Error type 5 at Line 9: unmatching types on both sides of assignment operator (=)
Error type 9 at Line 10: the function’s arguments mismatch the declared parameters
Error type 8 at Line 10: function return type is not matched
